docs/modules/ROOT/pages/reference/extensions/sql.adoc

```diff
@@ -67,6 +67,25 @@ Note that URI schemes such as `file` or `http` do not need to be listed.
 quarkus.camel.sql.script-files = queries.sql, classpath:sql/insert.sql
 ----
 
+=== SQL Aggregator ===
+
+If your exchanges in native mode contain objects other than:
+
+* `java.util.LinkedHashMap`
+* `java.util.HashMap`
+* `java.lang.String`
+* `java.lang.Number`
+* `java.lang.Integer`
```

Comment:
@ppalaga I used an approach, where I registered only the necessary objects to allow serialization of `DefaultExchangeHolder`.  (SQL extension then adds String and Integer)
Another approach would be to register basic types for serialization dorectly in core (like boolean, float, double, biginteger, long, may be some others).
Surely the second approach would be much more pleasant for the users.

If you agree, I would register basic types in core and would make this list bigger. 
From my POV it wold make sense to add it into some*.adoc here - near core - that these types are registered automatically and link these types from sql adoc.

Reply:
OK, now I understand much better how this was intended. Thanks for explaining!

I think that because this is going to be a temporary solution, we should not invest too much into it and we should try to avoid changes that we'll have to revert later thus risking regressions in end user apps. 

So I'd vote for moving the whole `serialization-config.json` to the test module and for leaving it up to the end users to maintain their `serialization-config.json` in their apps. Our SQL extension docs should contain a full listing of our `serialization-config.json` so that end users can copy&paste it to their repos. WDYT?

Reply:
@ppalaga  I agree, I refactored PR to make only minimal changes and full json is now part of the doc.